### ISSUE TITLE
NVDA does not interact with Huaway technology, Nokia PC Suite, and Symbion phones

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3896]
**Reported by jacobchawala on 2014-02-18 01:40**
when you connect the Nokia phone --say, Nokia N 72, or Huaway modems and other modems, to your computer, NVDA pronounces buttons and panes instead of reading its contents in those buttons and panes even if you press reading commands like tab and arrows. In N 72, NVDA cannot read messages. If you go to the contacts menu, it reads well all the contacts in such a way that you can edit them but not call them. In modems, it does nothing in such a way that we fail to connect our dongo modems after opening the software; instead, we connect by going to the system tray.

**Comment by leonardder** [comment]
@jacobchawala: are you the author of this issue? If so, could you provide a status update as whether this issue is still relevant for you? Note that Nokia PC Suite is deprecated software.

**Comment by Adriani90** [comment]
I suggest closing this as Nokia suite is not further developed anymore.
